mentat/parsers/change_display_helper.py

```diff
@@ -134,42 +137,42 @@ def get_full_change(display_information: DisplayInformation, prefix: str = ""):
             else ""
         ),
     ]
-    full_change = "\n".join([line for line in to_print if line])
-    prefixed_change = "\n".join(
-        (prefix + line) if line.strip() else line for line in full_change.split("\n")
-    )
-    return prefixed_change
+    for line in full_change:
+        if (isinstance(line, str) and line.strip()) or (
+            isinstance(line, Tuple) and line[0].strip()
```

Comment:
Is `line.strip()` in the if statement just checking if the line is empty or not? Might be more readable to do `line == ""` or `line != ""`

Reply:
We want to make sure it has stuff other than just whitespace; otherwise I would just do if line